### ISSUE TITLE
in-staff circulus, semicirculus and accentus

### ISSUE THREAD

**Issue by eroux** [gregorio-project/gregorio#1302]
Reported in [this thread](https://mail.gna.org/public/gregorio-users/2017-02/msg00021.html): it would be useful to have the possibility to get circulus, semi-circulus and accentus in-line, as in the example score pointed by the user:

![gregoriobug](https://cloud.githubusercontent.com/assets/60868/22924819/db3e2f9e-f2a6-11e6-9fe1-f08b4ef1868a.png)

I see two ways to do this:
- defining new codes for these variants (`r8`, `r9`, etc.)
- defining `\grechangesignplace{circulus}{instaff}`

I think I prefer the second one. It assumes that a score is consistent it its circulus, etc. placement, but it looks like a safe assertion...

In both cases, this means that the way the C code outputs circulus, etc. should be modified to output something similar to say horizontal episemus, although simpler, because we can assume that circulus, etc. are always above notes.

What do you think?

**Comment by henryso** [comment]
We should hold off on this until after version 5.  We also need to understand if these stack and how these stack along with the vertical and horizontal episemata.  Also, though you say we can assume it's above the note, once we implement it, someone will undoubtedly want it below the note.

**Comment by eroux** [comment]
You're right for delaying it to after 5.0, and also about the combination with episemata. That's the first request for such signs in 10 years, so I suppose they're quite rare and I'm quite sure noone will want that under a note (it would be pretty ugly), but you're right, if we can do it in a very clean and generic way, that would be best.

**Comment by upkl** [comment]
Since 5.0 is out—is there any news on this issue?  Those signs are used regularly in German chant (always above the note) and putting them above the staff looks rather ugly …

**Comment by henryso** [comment]
@upkl I'll look into this, but it won't make this year's TeXLive.  Do these shapes stack or do you only get one per note?

**Comment by upkl** [comment]
Thank you for answering to my request! AFAIK those signs are mutually exclusive:
circulus and semicirculus are used for stressed an unstressed syllables, respectively, and accentus is used in psalm tone models (as in the example above or in the following model):
![tzb-Ton-II](https://user-images.githubusercontent.com/2737384/75790543-a0dddd80-5d6b-11ea-98e6-7975e1083ec0.png)

**Comment by henryso** [comment]
How does this stack with the episemata?  Above?  Below? Configurable?  Mutually exclusive (doubtful)?  As I work through the code, this is starting to become somewhat complex.
